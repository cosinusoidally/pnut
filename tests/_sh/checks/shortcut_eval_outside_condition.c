// expect_comp_failure
void main() {
  int a = f() && g();
  int b = 1   && g();
  int c = f() && 1;
}
