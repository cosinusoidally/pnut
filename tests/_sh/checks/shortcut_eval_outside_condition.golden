tests/_sh/checks/shortcut_eval_outside_condition.c:7:0  comp_rvalue_go: && and || with function calls can only be used in tests
